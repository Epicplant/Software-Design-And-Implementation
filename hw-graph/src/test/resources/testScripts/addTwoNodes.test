# Adds 2 nodes to the graph

# Creates a new Graph
CreateGraph newGraph1

# Adds a new node "n1"
AddNode newGraph1 n1

# Adds a new node "n2"
AddNode newGraph1 n2

# Lists all of the nodes of this new graph
ListNodes newGraph1