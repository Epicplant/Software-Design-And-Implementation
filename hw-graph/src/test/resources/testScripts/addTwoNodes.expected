# Adds 2 nodes to the graph

# Creates a new Graph
created graph newGraph1

# Adds a new node "n1"
added node n1 to newGraph1

# Adds a new node "n2"
added node n2 to newGraph1

# Lists all of the nodes of this new graph
newGraph1 contains: n1 n2